pkg/liqo-controller-manager/resource-request-controller/offerUpdater.go

```diff
@@ -175,12 +136,42 @@ func (u *OfferUpdater) createOrUpdateOffer(cluster discoveryv1alpha1.ClusterIden
 		return true, err
 	}
 	klog.Infof("%s -> %s Offer: %s/%s", u.homeCluster.ClusterName, op, offer.Namespace, offer.Name)
-	return true, nil
+	return false, nil
 }
 
+// NotifyChange is used by the ResourceReader to notify that resources were added or removed.
+// It checks if any resources have changed by at least a set percentage since we last updated a ResourceOffer, and if so
+// it triggers a new update.
+func (u *OfferUpdater) NotifyChange() {
+	for clusterID := range u.currentResources {
```

Comment:
What about clusterIDs not yet in the map?

Reply:
The loop is meant to iterate over all clusters that we are offering resources to. The map is updated by `CreateOrUpdateOffer`, so there is no risk of "missing" cluster IDs I believe.